pkgs/development/tools/build-managers/gradle/tests/toolchains/src/main/java/Main.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("hello");
```

Comment:
How about:

```suggestion
        System.out.println(System.getProperty("java.version"));
```

and verify that in the test?